app/views/home/index.html.haml

```diff
@@ -1,6 +1,42 @@
-.jumbotron
-  %h1 Secret Santa
-  %p Welcome to Secret Santa
-  %p
-    = link_to "Create your first list!", new_user_registration_path, class: "btn btn-primary btn-lg"
-    = link_to "Already an organiser? Sign in", new_user_session_path, class: "btn btn-default btn-lg"
+%script{src:"//cdnjs.cloudflare.com/ajax/libs/Snowstorm/20131208/snowstorm-min.js"}
+.container
+  .col-md-4
+    %h1.b Secret Santa
+    %p Organise your secret santa gift exchange in minutes!
+    - unless current_user
+      %p= link_to "Create your first list!", new_user_registration_path, class: "btn btn-primary btn-lg btn-raised"
+      %p= link_to "Already an organiser? Sign in", new_user_session_path, class: "btn btn-default btn-lg btn-raised"
+    - else
+      %p= link_to "You're signed in! Go to Dashboard", member_dashboard_index_path, class: "btn btn-primary btn-lg btn-raised"
+  .col-md-8{style:"text-align:center;"}
+    = image_tag("santa_with_sack_full.png")
+.container
+  .row{style:"text-align:center;"}
+    .col-lg-4
+      %img.img-circle{height: "140", src: "assets/checklist.png", width: "140"}
+      %h3 Easy organising
+      %p{style:"text-align:justify;"}
+        If you're organising a gift exchange for your family or workplace, let
+        Secret Santa take care of the organising for you. Easily add lists of your
+        contacts, send out notification emails, and shuffle your list of santas.
+      %p
+        = link_to "Find out more", "#", class: "btn btn-primary btn-raised"
+    .col-lg-4
+      %img.img-circle{height: "140", src: "assets/open_letter.png", width: "140"}
```

Comment:
.h3.w3.br-100

If you hard code a height 140 it will probably look weird on a phone device